registrations/templates/message_to_signup.html

```diff
@@ -9,7 +9,12 @@
           <p style="color:#1a1a1a;font-size:16px;font-weight:normal;line-height:24px;margin:0px;">{{body |safe}}</p>
         </div>
         <div class="cta-button-wrapper" style="margin: 0px 0px 40px">
-          <a href="{{linked_registrations_ui_url}}/{{linked_registrations_ui_locale}}/registration/{{registration_id}}/enrolment/{{signup_id}}/edit" target="_blank" rel="noopener" class="cta-button" style="background:#0000bf;border-radius:0px;border:none;height:56px;text-align:center;text-decoration:none;vertical-align:middle;padding:16px 24px;box-sizing:border-box;display:inline-block;">
+        {% if signup.signup_group_id %}
+          <a href="{{linked_registrations_ui_url}}/{{linked_registrations_ui_locale}}/registration/{{registration_id}}/signup-group/{{signup.signup_group_id}}/edit"
+        {% else %}
+          <a href="{{linked_registrations_ui_url}}/{{linked_registrations_ui_locale}}/registration/{{registration_id}}/signup/{{signup.id}}/edit"
+        {% endif %}
+             target="_blank" rel="noopener" class="cta-button" style="background:#0000bf;border-radius:0px;border:none;height:56px;text-align:center;text-decoration:none;vertical-align:middle;padding:16px 24px;box-sizing:border-box;display:inline-block;">
```

Comment:
This is duplicated to several different email template. Maybe there could be a template for this part and import it here. 

See https://sonarcloud.io/component_measures?id=City-of-Helsinki_linkedevents&pullRequest=683&metric=new_duplicated_lines_density&view=list

Reply:
Addressed this too in the most recent changes. Went with a util function as didn't notice your comment.

Reply:
😄 That was fast. Util function is good as well.